packages/fxa-settings/src/components/ChooseNewsletters/en.ftl

```diff
@@ -4,11 +4,11 @@
 # Prompt above a checklist of newsletters
 choose-newsletters-prompt-2 = Get more from { -brand-mozilla }:
 # Newsletter checklist item
-choose-newsletters-option-take-action-for-the-internet-2 =
-  .label = Help keep the internet healthy
-# Newsletter checklist item
 choose-newsletters-option-security-privacy =
   .label = Security & privacy news and updates
 # Newsletter checklist item
 choose-newsletters-option-test-pilot =
   .label = Early access to test new products
+# Newsletter checklist item
```

Comment:
```suggestion
# Newsletter checklist item. This for a Mozilla Foundation newsletters, "Action alerts" can 
# be interpreted as "Calls to action"
```

I'm afraid "Action alerts" will not be easily translated without an explanation.

Reply:
Thanks! 😀